From bd5127130d4819df7644fcf03a09d12565805095 Mon Sep 17 00:00:00 2001
From: 5656565566 <contact@home56.top>
Date: Sun, 30 Aug 2026 14:39:50 +0800
Subject: [PATCH] sepolicy: label Floral WebView size properties

Change-Id: I99ae6002a589e351b230eb87036d0fe9a84ef1ea

diff --git a/private/property_contexts b/private/property_contexts
index a51fa3a07..da02897b8 100644
--- a/private/property_contexts
+++ b/private/property_contexts
@@ -721,6 +721,8 @@ ro.product.cpu.abi       u:object_r:build_prop:s0 exact string
 ro.product.cpu.abilist   u:object_r:build_prop:s0 exact string
 ro.product.cpu.abilist32 u:object_r:build_prop:s0 exact string
 ro.product.cpu.abilist64 u:object_r:build_prop:s0 exact string
+ro.floral.webview.vmsize32 u:object_r:system_prop:s0 exact int
+ro.floral.webview.vmsize64 u:object_r:system_prop:s0 exact int
 
 ro.product.system.brand        u:object_r:build_prop:s0 exact string
 ro.product.system.device       u:object_r:build_prop:s0 exact string
